setup.py

```diff
@@ -80,16 +95,6 @@ def get_cuda_bare_metal_version(cuda_dir):
         CppExtension('apex_C',
                      ['csrc/flatten_unflatten.cpp',]))
 
-def get_cuda_bare_metal_version(cuda_dir):
```

Comment:
Do we need to remove this function? The definition shouldn't hurt, even if it doesn't apply to ROCm? Just thinking from a point of view of minimizing diffs wrt upstream for future merges.

Reply:
I'll put it back to just help with later upstream merges, but for the record, the function appears twice as a copy/paste error.  I removed the latter copy.